src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -1110,115 +1109,16 @@ def _determine_ci_services(self, analyze_ctx: AnalyzeContext, git_service: BaseG
-
-                if build_tool.is_detected(analyze_ctx.component.repository.fs_path):
-                    logger.info("The repo uses %s build tool.", build_tool.name)
-                    analyze_ctx.dynamic_data["build_spec"]["tools"].append(build_tool)
-
-            if not analyze_ctx.dynamic_data["build_spec"]["tools"]:
-                logger.info(
-                    "Unable to discover any build tools for repository %s or the build tools are not supported.",
-                    analyze_ctx.component.repository.complete_name,
-                )
-
-            # Determine the CI services.
-            for ci_service in CI_SERVICES:
-                ci_service.load_defaults()
-                ci_service.set_api_client()
-
-                if ci_service.is_detected(
-                    repo_path=analyze_ctx.component.repository.fs_path,
-                    git_service=analyze_ctx.dynamic_data["git_service"],
-                ):
-                    logger.info("The repo uses %s CI service.", ci_service.name)
-
-                    # Parse configuration files and generate IRs.
-                    # Add the bash commands to the context object to be used by other checks.
-                    callgraph = ci_service.build_call_graph(
-                        analyze_ctx.component.repository.fs_path,
-                        os.path.relpath(analyze_ctx.component.repository.fs_path, analyze_ctx.output_dir),
-                    )
-                    analyze_ctx.dynamic_data["ci_services"].append(
-                        CIInfo(
-                            service=ci_service,
-                            callgraph=callgraph,
-                            provenance_assets=[],
-                            release={},
-                            provenances=[
-                                SLSAProvenanceData(
-                                    payload=InTotoV01Payload(statement=Provenance().payload),
-                                    asset=VirtualReleaseAsset(name="No_ASSET", url="NO_URL", size_in_bytes=0),
-                                )
-                            ],
-                        )
-                    )
-
-        # Determine the package registries.
-        # We match the software component against package registries through build tools.
         build_tools = (
             analyze_ctx.dynamic_data["build_spec"]["tools"] or analyze_ctx.dynamic_data["build_spec"]["purl_tools"]
         )
         for package_registry in PACKAGE_REGISTRIES:
             for build_tool in build_tools:
                 if package_registry.is_detected(build_tool):
                     analyze_ctx.dynamic_data["package_registries"].append(
-                        PackageRegistryInfo(
-                            build_tool=build_tool,
-                            package_registry=package_registry,
-                        )
+                        PackageRegistryInfo(build_tool=build_tool, package_registry=package_registry)
```

Comment:
nit: I wonder if we can add trailing comma to the list of parameters so that this diff can be removed? 
```suggestion
                        PackageRegistryInfo(
                            build_tool=build_tool,
                            package_registry=package_registry,
                        )
```